src/netex_idf/stops.rs

```diff
@@ -129,13 +130,73 @@ fn stop_point_parent_id(
         .map(|stop_area| stop_area.id.clone()))
 }
 
+#[derive(Eq, PartialEq, Hash, Clone, Copy)]
```

Comment:
In some of the files of `transit_model`, the structures are defined at the very beginning of the file (just after the `use` section). I'm not sure where I stand about that. Do you have any insight on the question to where these should be defined?

Reply:
no religion. I just wanted to modify the struct without scrolling.
If @ArnaudOggy or @patochectp whant to put at the beginning. I'll do it

Reply:
I tend to agree with woshilapin

Reply:
Funny, I don't really have a religion too :wink: and if I really had to choose, I think I would go with @datanel's idea. But I mentioned it because we already had this discussion with @ArnaudOggy :smile:

Reply:
done